src/webviews/webview-side/interactive-common/variableExplorerGrid.less

```diff
@@ -70,6 +70,7 @@
     color: var(--vscode-editor-foreground);
     border-style: none;
     padding: 0 2px;
+    font-family: monospace;
```

Comment:
Perhaps best disussed at standup or with UX, I'm not sure we want this or whether this is the right thing, 
this removes ability for users to costomize the fonts, however this is also required to fix the issue. Hence understand the need for something.

I.e. in effect this removes ability for users to use their own themes (fonts).

Reply:
(Replied here by mistake. I deleted my reply)